src/tokemize/cli.py

```
@@ -0,0 +1,208 @@
+        fn: Função a ser executada.
+        *args: Argumentos posicionais para a função.
+
+    Returns:
+        Resultado da função.
+
+    Raises:
+        typer.Exit: Com código 2 em caso de exceção.
+    """
+    try:
+        return fn(*args)
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)